File: C_Programming/Practise_Programs/PP_009.c
//////////////////////////////////////////////////////////////////
//
//  File Name    : Check_Even_Odd
//
//  Description  : This program accepts a number
//                 from the user and checks whether
//                 it is even or odd.
//
//  Example:
//      Input  : 11
//      Output : Number is Odd
//
//////////////////////////////////////////////////////////////////

//////////////////////////////////////////////////////////////////
//
//  Base Version: Check number Even or Odd.
//
//////////////////////////////////////////////////////////////////

/*
    START
        Accept number as No

        If No is completely divisible by 2
            then print Even

        Otherwise 
            print odd
    STOP


    START
        Accept number as No

        Divide No by 2

        If remainder is 0
            then print as Even

        Otherwise
            print as Odd
    STOP
*/

#include<stdio.h>

int main()
{
    int iValue = 0;
    int iRemainder = 0;

    printf("Enter number : \n");
    scanf("%d", &iValue);

    iRemainder = iValue % 2;

    if(iRemainder == 0)
    {
        printf("Number is Even\n");
    }
    else
    {
        printf("Number is Odd\n");
    }
    return 0;
}

File: Java_Programming/Practise_Programs/PP_009.java
//////////////////////////////////////////////////////////////////
//
//  File Name    : PP_009
//
//  Description  : This program accepts a number from the user 
//                 and checks whether it is divisible by both
//                 3 and 5 using a method that returns a
//                 boolean value.
//
//  Example:
//      Input  : 15
//      Output : Number is divisible by 3 and 5
//
//////////////////////////////////////////////////////////////////

// TYPE 2
import java.util.Scanner;

class PP_009
{
    public static boolean CheckDivisible(int iNo)
    {
        if((iNo % 3 == 0) && (iNo % 5 == 0))
        {
            return true;
        }
        else
        {
            return false;
        }           
    }

    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;
        boolean bRet = false;


        System.out.println("Enter number : ");
        iValue = sobj.nextInt(); 
        
        bRet = CheckDivisible(iValue);   
        
        if(bRet == true)
        {
            System.out.println("Number is divisible by 3 and 5");
        }
        else
        {
            System.out.println("Number is not divisible by 3 and 5");
        }
    }
}